ecmascript/transforms/module/src/util.rs

```diff
@@ -443,15 +438,6 @@ impl Scope {
         top_level: bool,
         expr: Expr,
     ) -> Expr {
-        macro_rules! entry {
-            ($i:expr) => {
-                folder
-                    .scope_mut()
-                    .exported_vars
-                    .entry(($i.sym.clone(), $i.span.ctxt()))
-            };
-        }
-
```

Comment:
This macro began raising an error that I did not understand, and I could not figure out how to fix it unless I stopped using the macro.  I think the error was caused by `folder.scope_mut()` returning a `RefMut<Scope>`.